refresh-transfer-list.user.js

```diff
@@ -17,11 +17,12 @@
   $(document).bind('DOMNodeInserted', function(event) {
     if ($(event.target).hasClass("SearchResults")) {
       if ($(event.target).find('#refreshList').length === 0) {
+        setInterval(function() { $(event.target).find('.pagingContainer').show(); }, 1000);
```

Comment:
Do we really need an interval for this?

Reply:
I have use setInterval because if i click more than once time on button "Refresh", the pagingContainer is hidden.

In the context i have only 1 page result